CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 * Adds support to partially push down tdigest aggregates
 
+* Supports binary encoded rslts from workers using citus.enable_binary_protocol
+
```

Comment:
I would like to write 
`Adds support for receiving binary encoded results from workers using citus.enable_binary_protocol` but it is too long, what about the above alternative @hanefi & @JelteF

Reply:
Just put it on two lines

Reply:
Don't abreviate results as rslts at least.

Reply:
afaik, we don't prefer multi line changelog entries, what do you think ?

Reply:
I thought in the last discussion the consensus was that our changelog was not very useful to users, because the items were too short. I think multiline entries are a good way to fix that, while keeping the 80 character line limit imposed by debian packaging.

Reply:
it totally makes sense, but I'm still not much sure as it is a decision more than this pr :/

Reply:
Some of our automated scripts can potentially break as they do not support multiline entries yet. However we can manually fix it this time.

This can also be a nice practice to figure out what needs to be updated on those scripts.

I'm ok with having a multiline entry here.

Reply:
addressed